bin/index.js

```diff
@@ -103,19 +113,26 @@ async function getFreePort(preferredPort = 5000) {
       }
     });
 }
-clear();
-const { open, port } = minimist(process.argv.slice(2), {
-  default: {
-    open: true,
-    port: 5000
-  }
-});
+
+/**
+ * Register Glicky commands
+ */
+glicky
+  .version(version, '-v, --version')
+  .usage('[options]')
+  .option('-o, --open <boolean>', 'Open Glicky in default browser', parseBoolean, true)
+  .option('-p, --port <port>', 'Open Glicky is specified port', parseInteger, 5000)
```

Comment:
should this be `<number>` instead of `port` as this is strictly a number type? We could then also change the description to 'Open Glicky on specified port number'